include/igl/kelvinlets.cpp

```diff
@@ -0,0 +1,208 @@
+// This file is part of libigl, a simple c++ geometry processing library.
+//
+// Copyright (C) 2013 Alec Jacobson <alecjacobson@gmail.com>
+//
+// This Source Code Form is subject to the terms of the Mozilla Public License
+// v. 2.0. If a copy of the MPL was not distributed with this file, You can
+
+#include "kelvinlets.h"
+#include "PI.h"
+#include "parallel_for.h"
+
+constexpr double POISSON_RATIO = 0.5;
+constexpr double SHEAR_MODULUS = 1;
+constexpr double a = 1 / (4 * igl::PI * SHEAR_MODULUS);
+constexpr double b = a / (4 * (1 - POISSON_RATIO));
+constexpr double c = 2 / (3 * a - 2 * b);
+
+template<typename Derivedx,
+         typename Derivedx0,
+         typename Derivedf,
+         typename DerivedF,
+         typename Scalar>
+IGL_INLINE auto kelvinlet_evaluator(const Scalar dt,
```

Comment:
Enclose within the `igl` namespace + maybe add some doc?

Reply:
Done